.gitignore

```diff
@@ -19,3 +19,4 @@
 .byebug_history
 
 /db/development_data
+/db/data/*
```

Comment:
In future versions, can you move your development data into a /db/development_data so we don't have to have two lines for ignore?